UPGRADE.md

```diff
@@ -1,5 +1,11 @@
 # Upgrade to 3.0
 
+## BC Break: Removed DBAL Connection factory from EntityManager
+
+EntityManager no longer provides a factory for DBAL Connection from an array.
+You must now pass an existing Connection instance directly.
+You can use `Doctrine\DBAL\DriverManager::getConnection()` factory to create new Connection.
```

Comment:
A better migration path is needed here. An example of before/after is needed.

Reply:
Updated.